go/cmd/generator/main.go

```diff
@@ -0,0 +1,150 @@
+// +build ignore
```

Comment:
@jcchavezs This would need to be ported for any language that we want to add config support for, right?

Reply:
Not really. This only applies for Golang as it is SDK instead of agent. See the reasoning here: https://github.com/hypertrace/agent-config/pull/1/files#diff-912bf9f2656a9118ef6ba0cb6677b10c712014e3b5b08f0a2e4fd865b92ee600R3